src/error.rs

```diff
@@ -205,6 +205,7 @@ impl std::ops::Deref for Error {
 }
 
 /// The types of errors that can occur.
+#[allow(missing_docs)]
```

Comment:
This suppresses warnings for missing comments on all of the `message` fields -- doesn't seems super necessary/helpful to users to have an individual comment for each of these fields given they're fairly self-explanatory.